code/game/machinery/spaceheater.dm

```diff
@@ -110,7 +115,7 @@
 
 	var/heat_capacity = enviroment.heat_capacity()
 	var/required_energy = abs(enviroment.temperature - target_temperature) * heat_capacity
-	required_energy = min(required_energy, heating_power * delta_time)
+	required_energy = min(required_energy, heating_power)
```

Comment:
ok so we're ~doubling heater output? that's good with me

Reply:
Ehh less i tested it and wasn't too strong, was pretty similar to the original one